feat(docs): add new page "Boxy" to learn section (#167)
codesandbox/docs: packages/projects-docs/pages/learn/boxy/getting-started.mdx

---
title: Getting Started
description: Instructions for getting started with AI
---

# Getting Started

> All AI tools are available for Pro users. In order to enable these features, admins will need to grant permissions in the workspace settings.  

Reviewer: ```suggestion
All CodeSandbox AI tools are available for Pro users. To enable these features, admins need to grant permissions in the workspace settings.
```

<!-- todo: insert image -->

> Once you grant AI permissions, CodeSandbox will send information about the event to OpenAI for analysis. Under all circumstance we will not send any event data to OpenAI without an explicit instruction to do so. Only by toggling permissions on will selected event data be shared with OpenAI. OpenAI does not use the event data shared via this feature to train their AI models. To learn more about our privacy agreement with OpenAI, check out our [privacy page](https://codesandbox.io/legal/privacy)
> 
> ## Running AI
> Once permissions have been granted, you will need to restart the VMs for the changes to take effect. You can do this by selecting `Restart Branch` in the main menu. If you are in a cloud sandbox, the option will read `Restart Sandbox`.
> 
> 
> ## Boxy for Sandboxes
> Boxy is available only on repositories and cloud sandboxes. If you want the support of Boxy in your browser sandbox, you can easily convert your sandbox using the options in the toolbar. 

Reviewer: ```suggestion
Once you grant AI permissions, CodeSandbox will send information about the event to OpenAI for analysis. We will never, under any circumstances, send any event data to OpenAI without explicit instruction to do so. We will only share selected event data with OpenAI after you toggle on these permissions. OpenAI does not use the event data shared via this feature to train its AI models. To learn more about our privacy agreement with OpenAI, check out our [Privacy page](https://codesandbox.io/legal/privacy).

## Running AI
Once permissions have been granted, you need to restart the VMs for the changes to take effect. You can do this by selecting `Restart Branch` in the main menu. If you are in a cloud sandbox, the option will read `Restart Sandbox`.		

## Boxy for Sandboxes
Boxy is available only on repositories and cloud sandboxes. If you want the support of Boxy in your browser sandbox, you can easily [convert your sandbox](/tutorial/convert-browser-sandbox-cloud) using the options in the toolbar.
```
